### ISSUE TITLE
Videos aren't playing in amp-twitter

### ISSUE THREAD

**Issue by Enriqe** [ampproject/amphtml#27938]
It's not possible to play videos inside `amp-twitter`.

Steps to repro:
1. Go to an AMP page with the `amp-twitter` component that has a video. e.g. https://amp.dev/documentation/components/amp-twitter/?format=websites
2. Click on the video

Expected: 
Video plays

Actual:
Blank frame with play button

Error logs:

`TwitterVideoPlayerIframe.1c7d5c71349918c3.js:1 POST https://api.twitter.com/1.1/guest/activate.json 403`

```
{
  errors: [
    {
      code: 215,
      message: "Bad Authentication data."
    }
  ]
}
```

/to @ampproject/wg-ui-and-a11y

**Comment by caroqliu** [comment]
Not able to reproduce on RTV 2004172112280, Chrome 81 :/ Clicking on the video thumbnnail leads to the iframe flashing white then reappearing with the thumbnail, clicking a second time causes the video to play. Not great but not seeing any authentication errors. Am I doing something wrong here? 

The video I'm looking at is the [sample under `Behavior`](https://amp.dev/documentation/components/amp-twitter/?format=websites#behavior).

**Comment by Enriqe** [comment]
Thanks for looking into this @caroqliu !

Huh weird... It seems like deleting my cookies coming from twitter.com and trying again solved it for me. /cc @gmajoulet who also has this issue

Closing

**Comment by gmajoulet** [comment]
It fails in non private navigation (same issue as Enrique) but works in private nav. I'm not deleting my cookies in case we need someone who can repro :))
Could it be because of an ad blocker?
